doc/gettingStarted/Applications.md

```diff
@@ -0,0 +1,102 @@
+- Ajust how long the timer should go for with the time picker.
+- Press the Start button at the bottom to start/stop the timer.
+
+### Steps
+![Steps UI](/doc/gettingStarted/AppsScreenshots/steps.png)
+- The total count of steps for the current display will show in the middle of the screen.
+- The Reset button in the bottom middle resets the Trip counter. (Total of all steps taken.)
+- The progress circle shows the percentage of your daily goal completed.
+
+### Heartrate
+![Heartrate UI](/doc/gettingStarted/AppsScreenshots/Heartrate.png)
+- Press Start to start measuring your heartrate.
+    - It may take a bit to get the first measurement.
+    - The monitoring currently does not happen while your screen is off, however the screen will not go to sleep while the heartrate app is open.
+
+### Music
+![Music UI](/doc/gettingStarted/AppsScreenshots/Music.png)
+- This app shows currently playing music.
+    - Please note that this app is not very useful without a device connected.
+- Press the button in the center to play/pause, and the buttons on the left and right to go to the previous and next tracks, respectively.
+- Swipe up to get to volume controls.
+
+### InfiniPaint
+![InfiniPaint UI](/doc/gettingStarted/AppsScreenshots/InfiniPaint.png)
+- This app does not allow you to swipe from the top to exit, use the side button instead.
+- Draw on the screen to add lines.
+- Hold down in one spot to change paint colors.
+
+### Pong
+![Pong UI](/doc/gettingStarted/AppsScreenshots/Pong.png)
+- This app does not allow you to swipe from the top to exit, use the side button instead.
+- Drag your finger to move the paddle.
+- Goal: Don't let the ball go off the left side of the screen.
+
+### 2048
+![2048 UI](/doc/gettingStarted/AppsScreenshots/2048.png)
+- This app does not allow you to swipe from the top to exit, use the side button instead.
+- Play a game of 2048.
+- Swipe up, down, left, or right tomove the tiles.
+- When two tiles with the same number run into each other, they will add together.
+- Goal: Don't let the screen fill up with tiles, and get to the 2048 tile to win.
+
+### Dice
+![Dice UI](/doc/gettingStarted/AppsScreenshots/Dice.png)
+- This app is not in InfiniTime 1.14.
+- Ajust the count to change the number of dice.
+- Ajust the sides to change the number of sides.
+- Press the button at the bottom to roll the die.
+- The result will be on the right side of the screen.
+
+### Metronome
+![Metronome UI](/doc/gettingStarted/AppsScreenshots/Metronome.png)
+- Ajust the BPM with the circular slider.
+    - A bug currently makes it always snap to 98 BPM.
+- Use the button in the bottom left to start the metronome.
+
+### Maps
+![Maps UI](/doc/gettingStarted/AppsScreenshots/Maps.png)
+- This app shows info from a navigation app.
+    - Please note that this app is not very useful without a device connected.
+
+### Weather
+![Weather UI](/doc/gettingStarted/AppsScreenshots/Weather.png)
+- This app is not in InfiniTime 1.14.
```

Comment:
This line is also probably not needed if this PR is supposed to be used for 1.15.0

Reply:
Done!